storage/src/main/java/org/apache/kafka/storage/internals/log/RemoteIndexCache.java

```diff
@@ -669,4 +689,10 @@ public static String remoteTransactionIndexFileName(RemoteLogSegmentMetadata rem
         return generateFileNamePrefixForIndex(remoteLogSegmentMetadata) + LogFileUtils.TXN_INDEX_FILE_SUFFIX;
     }
 
+    public static int estimatedEntrySize(Entry entry) {
+        return entry.offsetIndex.sizeInBytes() + (int) entry.offsetIndex.length() +
```

Comment:
why are we double counting offset and time index?

Reply:
Sorry, I mistook mmap and file as two different things.